src/cluster/utils.go

```diff
@@ -10,30 +10,30 @@ import (
 	"github.com/spf13/viper"
 )
 
-//generateUniqueID returns a unique id for assigning to a new microcontroller
+//generateUniqueID returns a unique id for assigning to a new device
 func (c Cluster) generateUniqueID() int {
```

Comment:
This doesnt generate a unique ID. Just a random one within the range of `MICROCONTORLLER_LIMIT`.

Also I imagined the unique ID could quite literally be something unique to each physical board. Like its MAC address or a hash of it or something. That way there is no dependency on the device to remember its config. It can turn on and the DeviceCommander can auto associate past configurations with it.

Reply:
this is dead code that was missed when refactoring gofire. There I believe it was a placeholder for a database or some future unique id concept. DeviceCommander will use the serial number from the physical board like it says it in the spec 👍